web/groups/hlt.json

```
@@ -478,6 +477,8 @@
   "MaskedMeasurementTrackerEventProducer|hltIter?L3Muon*": "Muons",
   "MaskedMeasurementTrackerEventProducer|": "Tracking",
   "MeasurementTrackerEventProducer|": "Tracking",
+  "MergeClusterProducer|hltHgcalMergeLayerClusters*": "HGCal",
+  "MergeClusterProducer|hltMergeLayerClusters*": "HGCal",
```

could it be simply
```suggestion
  "MergeClusterProducer|": "HGCal",
```
?

Or do we foresee having `MergeClusterProducer`s that belong to different groups ?

---

> Or do we foresee having MergeClusterProducers that belong to different groups ?

This possibility was indeed discussed, but I have no strong opinion. Either was is fine by me.